source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionCanBeCancelled.cs

```diff
@@ -65,14 +67,14 @@ public async Task DuringGetCapabilities_ScriptExecutionCanBeCancelled(TentacleTy
                             {
                                 if (rpcCall == RpcCall.RetryingCall && capabilityServiceV2Exceptions.GetCapabilitiesLatestException == null)
                                 {
-                                    service.EnsureTentacleIsConnectedToServer(Logger);
```

Comment:
As responses are cached, we can't make a call to the ICapabilitesService to ensure tentacle is connected but need to use another service.

Reply:
Not a big deal, but is there a way to make this more obvious so the next person doesn't have the bright idea of "using `service` so we don't need `scriptServiceV2`"?

Reply:
I have no good ideas for this so leaving as is for now.